### PR TITLE
[Overview] 🐛🌐 Fix AU police motorbike training

### PR DESCRIPTION
**Pull request by Gooochy**

### CODE REVIEW

File: src/i18n/en_AU.ts [hunk 1]
```diff
@@ -1148,9 +1148,9 @@ export default {
                 staffList: 'K-9',
             },
             {
-                caption: 'Police Motorcycle',
+                caption: 'Motor Officer',
```
Comment (Suchty112) [inline src/i18n/en_AU.ts:1151]: this is wrong here, the way it was before is right. 
The caption that appears in the school and in the course window of the alliance must be placed here.
https://cdn.discordapp.com/attachments/544869019673559060/918244105979588698/unknown.png

File: src/i18n/en_AU.ts [hunk 2]
```diff
@@ -348,7 +348,7 @@ export default {
             possibleBuildings: [6, 19],
             schooling: {
                 Police: {
-                    'Police Motorcycle': {
+                    'Motor Officer': {
```
Comment (Suchty112) [inline src/i18n/en_AU.ts:351]: this is also wrong, here it must be exactly the same as in the other picture or in this picture.
https://cdn.discordapp.com/attachments/544869019673559060/918244105778253914/unknown.png

File: src/i18n/en_AU.ts [hunk 3]
```diff
@@ -1148,9 +1148,9 @@ export default {
                 staffList: 'K-9',
             },
             {
-                caption: 'Police Motorcycle',
+                caption: 'Motor Officer',
                 duration: '3 Days',
-                staffList: 'Police Motorcycle',
+                staffList: 'Motor Officer',
```
Comment (Suchty112) [inline src/i18n/en_AU.ts:1153]: this should be exceptionally correct

**Review by Suchty112 — CHANGES_REQUESTED** [review]

**Comment by Suchty112** [comment]
please wait with any changes, i will do a debugbuild

**Comment by Suchty112** [comment]
We are leaving this PR in the draft until Jan looks over it. the problem with the motorbike is that it has 2 different names for the training.
Police Motorcycle at the school/in the class overview in the school/association overview or in the assignment window and Motor Officer when it has been trained.

**Comment by JRH-1997** [comment]
That's the same in NL: 
https://github.com/LSS-Manager/LSSM-V.4/blob/f8048f47e25e14ef07425b6126f583f3544687c0/src/i18n/nl_NL.ts#L1669

stafflist when it's trained, the other one in the school or when its still in training

**Comment by Gooochy** [comment]
Yeah the reason I did it is in the add a vehicle tab it says motor officer

**Review by Suchty112 — APPROVED** [review]

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**